### PR TITLE
fix: Make title of manual pages more descriptive

### PR DESCRIPTION
**Pull request by hirasawayuki**
resolve https://github.com/denoland/deno_website2/issues/1766

In this PR, all Manual pages were labeled `Manual | Deno`, so the titles of the Manual pages were made more descriptive.

When the state of the sourceURL is changed, the title of the page is also changed.

The attached image is the browser history.
before | after
---- | ----
before | after

### CODE REVIEW

**Comment by bartlomieju** [comment]
A nitpick from me: maybe `{title} | Manual | Deno` would be better to clearly signal that this is a manual page?

**Comment by hirasawayuki** [comment]
I'll fix it.

**Comment by hirasawayuki** [comment]
@bartlomieju 
Thank you for your suggestion. I have fixed it!

**Comment by bartlomieju** [comment]
Thanks! I'll wait for Luca's review before merging.

**Review by lucacasonato — COMMENTED** [review]
Can we instead use the page title from the table of contents?

**Comment by hirasawayuki** [comment]
@lucacasonato 
Thanks for the review!
I fixed it to get the page title from the table of contents.

File: util/manual_utils.test.ts
Comment (kt3k) [inline util/manual_utils.test.ts:38]: :+1:

**Review by kt3k — APPROVED** [review]
Nice work! LGTM. Thank you for the contribution @hirasawayuki

**Pull request merged**
